src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -2,18 +2,30 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Lock;
+import org.springframework.data.jpa.repository.Query;
 
+import java.sql.Timestamp;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.stream.Stream;
 import javax.persistence.LockModeType;
 
 public interface LetterRepository extends JpaRepository<Letter, UUID> {
+
+    String STALE_LETTERS_QUERY = "select l "
```

Comment:
`select l`?

Reply:
because `from Letter l`. entity based table definition. alias is typical - first letter

Reply:
does it select all the columns defined in `Letter` entity class? We only want the ones from `StaleLetter`, right?

Reply:
oh. i though magic is quite strong on projections too. unfortunately, tried, it does select all columns